### PR TITLE
Alpha 3 cleanup

### PR DESCRIPTION
**Pull request by cpvalente**
small lint cleanups

@alex-Arc, the logic around spreadsheets is a bit heavy and I thought we could slowly try to simplify, can you sanity check the change?

### CODE REVIEW

**Review by alex-Arc — APPROVED** [review]

File: apps/server/src/services/sheet-service/sheetUtils.ts [hunk 1]
```diff
@@ -74,7 +74,6 @@ export function cellRequestFromEvent(
   worksheetId: number,
   metadata,
 ): sheets_v4.Schema$Request {
-  const returnRows: sheets_v4.Schema$CellData[] = [];
   const tmp = Object.entries(metadata)
```
Comment (cpvalente) [inline apps/server/src/services/sheet-service/sheetUtils.ts:77]: @alex-Arc can you help me find a better name for this? What is in this array?

File: apps/server/src/services/sheet-service/sheetUtils.ts [hunk 2]
```diff
@@ -146,3 +112,36 @@ export function cellRequestFromEvent(
     },
   };
 }
+
+function getCellData(key: string, event: OntimeRundownEntry) {
+  if (isOntimeEvent(event)) {
+    if (key === 'blank') {
+      return {};
+    }
+    if (key === 'blank') {
```
Comment (cpvalente) [inline apps/server/src/services/sheet-service/sheetUtils.ts:121]: Note to self: I think this is supposed to say colour

File: apps/server/src/services/sheet-service/sheetUtils.ts (hunk 1 above)
Comment (alex-Arc) [reply apps/server/src/services/sheet-service/sheetUtils.ts:77]: it could be rowData as it is just an array of cellData

Comment (cpvalente) [reply apps/server/src/services/sheet-service/sheetUtils.ts:77]: Makes sense. And what is the filtering doing?

Comment (alex-Arc) [reply apps/server/src/services/sheet-service/sheetUtils.ts:77]: I think it takes out all the stuff that was not found in the sheet
so if there was no duration in the sheet it is removed here so it dose not become part of the upload
it is then sorted so it lands in the correct order

**Pull request merged**